docs/configuration/integrations/integrations-next/app-o11y-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_receiver_config
+
+The `app_o11y_receiver_config` block configures the `app-o11y-receiver`
+integration. This integration exposes a http endpoint that can receive telemetry
+from the [grafana javascript agent](https://github.com/grafana/grafana-javascript-agent)
+and forward it to logs, traces or metrics backends.
+
+Full reference of options:
+
+```yaml
+  autoscrape:
+    # Enables autoscrape of integrations.
+    [enable: <boolean> | default = true]
+
+    # Specifies the metrics instance name to send metrics to. Instance
+    # names are located at metrics.configs[].name from the top-level config.
+    # The instance must exist.
+    #
+    # As it is common to use the name "default" for your primary instance,
+    # we assume the same here.
+    [metrics_instance: <string> | default = "default"]
+
+    # Autoscrape interval and timeout. Defaults are inherited from the global
+    # section of the top-level metrics config.
+    [scrape_interval: <duration> | default = <metrics.global.scrape_interval>]
+    [scrape_timeout: <duration> | default = <metrics.global.scrape_timeout>]
+
+  # Integration instance name
+  [instance: <string>]
+
+  # Domains in which the agent is sending data from. For example "https://myapp.com"
+  cors_allowed_origins:
+    [- <string>]
+
+  # Configure rate limiting. The HTTP server of the App observability implements
+  # a token bucket rate limitng algorithm in which we can configure the maximum RPS
+  # as well as the burstiness (peaks of RPS)
+  rate_limiting:
+    [enabled: <boolean> | default = false]
+    [rps: <number> | default = 100]
+    [burstiness: <number> | default = 50]
+
+  # If configured, incoming requests will be required to specify this key in "x-api-key" header
+  [api_key: <string>]
```

Comment:
naming nit: maybe this should be `validate_api_key` so it's clearer to users that we're talking about API keys to check for and not an API key that the receiver uses? 

Also, we should support a `_file`-based form of this so secrets aren't stored in the config file directly.